added new solutions, will add commentary later

tinyfold88 opened the pull request.

AStrangePotato commented:
Looks good. Nice solutions.

Tankman61 commented:
By the way Tinyfold, would you like to add yourself to the contributors
list in the about section?

On Wed, Apr 23, 2025 at 5:30 PM AStrangePotato ***@***.***>
wrote:

> Merged #15 <https://github.com/Tankman61/CCCSolutions/pull/15> into main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tankman61/CCCSolutions/pull/15#event-17378529978>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJRPDNBOMAD2A2YR2KPSSCD23AA75AVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJXGM3TQNJSHE4TOOA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Tankman61 commented:
Oh sorry. We changed the sol availability from lazy loading to just being
on the constants.js file. I believe contributors are also in that file you
can add a row there

On Thu, Apr 24, 2025 at 11:40 AM tinyfold88 ***@***.***>
wrote:

> *tinyfold88* left a comment (Tankman61/CCCSolutions#15)
> <https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828084884>
> Sure, I can figure it out when I have time. I see that the website has
> <br> updated, but not the solution column. Could you show me where to fix
> that? <br> Also, maybe we could rename that column to &quot;solution
> available&quot;. <br> Best regards, <br> Tinyfold <br> [image: image.png]
> <br> <br> On Thu, Apr 24, 2025 at 10:30 AM William Yang ***@***.***&gt;
> <br> wrote: <br> <br> &gt; *Tankman61* left a comment
> (Tankman61/CCCSolutions#15) <br> &gt; &lt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2827863184&gt;
> <br> &gt; By the way Tinyfold, would you like to add yourself to the
> contributors <br> &gt; &lt;br&gt; list in the about section? &lt;br&gt;
> &lt;br&gt; On Wed, Apr 23, 2025 at 5:30 PM <br> &gt; AStrangePotato
> ***@***.***&amp;gt; &lt;br&gt; wrote: &lt;br&gt; &lt;br&gt; &amp;gt; Merged
> #15 &amp;lt; <br> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15&amp;gt; into main.
> &lt;br&gt; <br> &gt; &amp;gt; &lt;br&gt; &amp;gt; — &lt;br&gt; &amp;gt;
> Reply to this email directly, view it on GitHub <br> &gt; &lt;br&gt;
> &amp;gt; &amp;lt; <br> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15#event-17378529978&amp;gt;,
> <br> &gt; or &lt;br&gt; &amp;gt; unsubscribe &lt;br&gt; &amp;gt; &amp;lt;
> <br> &gt;
> https://github.com/notifications/unsubscribe-auth/BJRPDNBOMAD2A2YR2KPSSCD23AA75AVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJXGM3TQNJSHE4TOOA&amp;gt;
> <br> &gt; &lt;br&gt; &amp;gt; . &lt;br&gt; &amp;gt; You are receiving this
> because you are subscribed to <br> &gt; this thread.Message &lt;br&gt;
> &amp;gt; ID: ***@***.***&amp;gt; &lt;br&gt; &amp;gt; &lt;br&gt; <br> &gt;
> <br> &gt; — <br> &gt; Reply to this email directly, view it on GitHub <br>
> &gt; &lt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2827863184&gt;,
> <br> &gt; or unsubscribe <br> &gt; &lt;
> https://github.com/notifications/unsubscribe-auth/AXUOC4OSRZCOUXT2XD4LXBT23DYRZAVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRXHA3DGMJYGQ&gt;
> <br> &gt; . <br> &gt; You are receiving this because you authored the
> thread.Message ID: <br> &gt; ***@***.***&gt; <br> &gt; <br>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828084884>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJRPDNFFPS6HD25O76Z67C323EAYBAVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRYGA4DIOBYGQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

tinyfold88 commented:
Ok, got it. I will go implement

On Thu, Apr 24, 2025 at 1:12 PM William Yang ***@***.***>
wrote:

> *Tankman61* left a comment (Tankman61/CCCSolutions#15)
> <https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828312673>
> Oh sorry. We changed the sol availability from lazy loading to just being
> <br> on the constants.js file. I believe contributors are also in that file
> you <br> can add a row there <br> <br> On Thu, Apr 24, 2025 at 11:40 AM
> tinyfold88 ***@***.***&gt; <br> wrote: <br> <br> &gt; *tinyfold88* left a
> comment (Tankman61/CCCSolutions#15) <br> &gt; &lt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828084884&gt;
> <br> &gt; Sure, I can figure it out when I have time. I see that the
> website has <br> &gt; &lt;br&gt; updated, but not the solution column.
> Could you show me where to fix <br> &gt; that? &lt;br&gt; Also, maybe we
> could rename that column to &amp;quot;solution <br> &gt;
> available&amp;quot;. &lt;br&gt; Best regards, &lt;br&gt; Tinyfold
> &lt;br&gt; [image: image.png] <br> &gt; &lt;br&gt; &lt;br&gt; On Thu, Apr
> 24, 2025 at 10:30 AM William Yang ***@***.***&amp;gt; <br> &gt; &lt;br&gt;
> wrote: &lt;br&gt; &lt;br&gt; &amp;gt; *Tankman61* left a comment <br> &gt;
> (Tankman61/CCCSolutions#15) &lt;br&gt; &amp;gt; &amp;lt; <br> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2827863184&amp;gt;
> <br> &gt; &lt;br&gt; &amp;gt; By the way Tinyfold, would you like to add
> yourself to the <br> &gt; contributors &lt;br&gt; &amp;gt;
> &amp;lt;br&amp;gt; list in the about section? &amp;lt;br&amp;gt; <br> &gt;
> &amp;lt;br&amp;gt; On Wed, Apr 23, 2025 at 5:30 PM &lt;br&gt; &amp;gt;
> AStrangePotato <br> &gt; ***@***.***&amp;amp;gt; &amp;lt;br&amp;gt; wrote:
> &amp;lt;br&amp;gt; &amp;lt;br&amp;gt; &amp;amp;gt; Merged <br> &gt; #15
> &amp;amp;lt; &lt;br&gt; &amp;gt; <br> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15&amp;amp;gt; into main.
> <br> &gt; &amp;lt;br&amp;gt; &lt;br&gt; &amp;gt; &amp;amp;gt;
> &amp;lt;br&amp;gt; &amp;amp;gt; — &amp;lt;br&amp;gt; &amp;amp;gt; <br> &gt;
> Reply to this email directly, view it on GitHub &lt;br&gt; &amp;gt;
> &amp;lt;br&amp;gt; <br> &gt; &amp;amp;gt; &amp;amp;lt; &lt;br&gt; &amp;gt;
> <br> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15#event-17378529978&amp;amp;gt;,
> <br> &gt; &lt;br&gt; &amp;gt; or &amp;lt;br&amp;gt; &amp;amp;gt;
> unsubscribe &amp;lt;br&amp;gt; &amp;amp;gt; &amp;amp;lt; <br> &gt;
> &lt;br&gt; &amp;gt; <br> &gt;
> https://github.com/notifications/unsubscribe-auth/BJRPDNBOMAD2A2YR2KPSSCD23AA75AVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJXGM3TQNJSHE4TOOA&amp;amp;gt;
> <br> &gt; &lt;br&gt; &amp;gt; &amp;lt;br&amp;gt; &amp;amp;gt; .
> &amp;lt;br&amp;gt; &amp;amp;gt; You are receiving this <br> &gt; because
> you are subscribed to &lt;br&gt; &amp;gt; this thread.Message
> &amp;lt;br&amp;gt; <br> &gt; &amp;amp;gt; ID: ***@***.***&amp;amp;gt;
> &amp;lt;br&amp;gt; &amp;amp;gt; &amp;lt;br&amp;gt; &lt;br&gt; &amp;gt; <br>
> &gt; &lt;br&gt; &amp;gt; — &lt;br&gt; &amp;gt; Reply to this email
> directly, view it on GitHub &lt;br&gt; <br> &gt; &amp;gt; &amp;lt; <br>
> &gt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2827863184&amp;gt;,
> <br> &gt; &lt;br&gt; &amp;gt; or unsubscribe &lt;br&gt; &amp;gt; &amp;lt;
> <br> &gt;
> https://github.com/notifications/unsubscribe-auth/AXUOC4OSRZCOUXT2XD4LXBT23DYRZAVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRXHA3DGMJYGQ&amp;gt;
> <br> &gt; &lt;br&gt; &amp;gt; . &lt;br&gt; &amp;gt; You are receiving this
> because you authored the <br> &gt; thread.Message ID: &lt;br&gt; &amp;gt;
> ***@***.***&amp;gt; &lt;br&gt; &amp;gt; &lt;br&gt; <br> &gt; <br> &gt; —
> <br> &gt; Reply to this email directly, view it on GitHub <br> &gt; &lt;
> https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828084884&gt;,
> <br> &gt; or unsubscribe <br> &gt; &lt;
> https://github.com/notifications/unsubscribe-auth/BJRPDNFFPS6HD25O76Z67C323EAYBAVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRYGA4DIOBYGQ&gt;
> <br> &gt; . <br> &gt; You are receiving this because you commented.Message
> ID: <br> &gt; ***@***.***&gt; <br> &gt; <br>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tankman61/CCCSolutions/pull/15#issuecomment-2828312673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXUOC4IINEAUOYKNULRJEA323ELO7AVCNFSM6AAAAAB3XEPDNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRYGMYTENRXGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
